### PR TITLE
feat(onepane): Added a quickstart for onepane

### PR DESCRIPTION
**Pull request by sajosam**
# Summary

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

_The owners of this repo are not experts in the subject matter of the quickstarts. We review for the quickstart to be functional and for security risks. If you are seeking feedback on the content of the quickstart, please seek out a subject matter expert. If you are not an internal NR contributor, we can do our best to connect you with a content reviewer._

## Pre merge checklist

- [x] Did you check you NRQL syntax? - Does it work?
- [x] Did you include a Data source and Documentation reference?
- [x] Are all documentation links publicly accessible?
- [x] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [x] Did you check your descriptive content for spelling and grammar errors?
- [x] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### CODE REVIEW

**Comment by jcountsNR** [comment]
@sajosam Thanks for the submission! I think there are a few points with this one.

1. There should still be a quickstart defined in the quickstarts directory that points to this data-source.
2. The longer description of 'what is onepane' could be moved there, since the datasource description would be fit into a very small tile. The datasource description should be very minimal, something like `connect OnePane change tracking to New Relic`.

The second point is also why the submission is failing. Once you move the longer description over to a quickstart, then it wouldn't have this issue.

File: quickstarts/onepane/config.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+keywords:
+  - devops
+  - change tracking
+  - incident correlation
+  - root cause analysis
+  - llm
+
+documentation:
+  - name: Installation docs
+    url: dhttps://www.onepane.ai/docs/en/articles/8999270-adding-onepane-change-tracking-to-your-new-relic-deployment
```
Comment (mdumpati) [inline quickstarts/onepane/config.yml:43]:
```suggestion
    url: https://www.onepane.ai/docs/en/articles/8999270-adding-onepane-change-tracking-to-your-new-relic-deployment
```

File: quickstarts/onepane/config.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+id: 00000-0000-0000-0000000
```
Comment (RamanaReddy8801) [inline quickstarts/onepane/config.yml:1]:
```suggestion

```

**Review by mdumpati — COMMENTED** [review]
@sajosam enter correct url?

**Review by RamanaReddy8801 — CHANGES_REQUESTED** [review]

**Comment by sajosam** [comment]
Thanks for the feedback @RamanaReddy8801! 
The issue has been resolved.

**Review by RamanaReddy8801 — APPROVED** [review]
LGTM!

**Pull request merged**